actions/setup_e2e_tests.sh

```diff
@@ -12,10 +12,24 @@ if [[ -n "$RHTEST" ]]; then
     RHVERSION=`cat /etc/redhat-release 2> /dev/null | sed -r 's/([^0-9]*([0-9]*)){1}.*/\2/'`
     echo "*** Detected Distro is ${RHTEST} - ${RHVERSION} ***"
     sudo yum install -y python-pip wget
+    if [[ "$RHVERSION" -ge 7 ]]; then
+        sudo yum install -y bats jq
+    else
+        # Install from GitHub
+        git clone https://github.com/bats-core/bats-core.git
+        (cd bats-core; sudo ./install.sh /usr/local)
+    fi
 elif [[ -n "$DEBTEST" ]]; then
-    echo "*** Detected Distro is ${DEBTEST} ***"
-    sudo apt-get install -y wget
-    sudo apt-get -q -y install python-pip python-dev build-essential
+    DEBVERSION=`lsb_release --release | awk '{ print $2 }'`
+    echo "*** Detected Distro is ${DEBTEST} - ${DEBVERSION} ***"
+    sudo apt-get -q -y install build-essential jq python-pip python-dev python3-venv wget
```

Comment:
Just curious - why do we need python3 here?

Reply:
We don't - I'll remove it. I originally put it in to test on Ubuntu 14/Cent OS 6 but we're skipping those so I don't need it.